src/views/BlogPostPage/BlogPostPage.js

```diff
@@ -27,6 +27,8 @@ export default function BlogPostPage() {
   const alanKey = process.env.REACT_APP_ALAN_KEY;
 
   const askButtonHandler = () => {
+    // Check is a instance of alanBtn is running
+    if (alanBtn().isActive()) return;
```

Comment:
It's still happening.
By calling `alanBtn()` you're creating a new instance again. Your changes are deployed in the below link. Please take a look.

https://5f8d5c3466a6d10008285489--timesofinternet.netlify.app/

Reply:
Okay, I tried it again. 

I created an instance of alanBtn in react use effect and moved all your previous codes. 
That way I used the instance anytime alanBtn is needed.